01_02_Combat_Rules.txt

```diff
@@ -444,25 +444,35 @@ You can jump vertically Str/3 meters.
 ===== Other Combat Details =====
 
 Critical Hits:
-	Critical Hits are accuracy rolls that land inside your Critical Hit range 
-(aka. critical chance). The base critical hit range is 1, meaning an Accuracy roll of 
-10 is a critical hit. Luck modifies your critical hit range: (Luck Modifier/10), round 
-down, is added to your critical hit range. Therefore if you have a Luck stat 
-of 8, corresponding to a Luck Modifier of 12, then you would have +1 to your 
-critical hit range and would critically hit on rolls of 9-10. Negative Luck modifiers 
-do not reduce Critical Hit range.
-
-	When your Accuracy roll lands in the upper half of your Critical Hit range,
-it is referred to as an Extra-Critical Hit. When the range is odd, it may be necessary 
-to roll a confirmation D10 to determine which kind of crit you score. Example: If you 
-have a Critical range of 1 and roll a 10, you would roll another D10 and score a 
-Critical hit on 1-5 or an Extra-Critical on 6-10.
-
-	Regular Critical Hits deal +20 damage, while Extra-Criticals deal +30 damage. 
-Some abilities or feats have additional effects that trigger when you score Criticals 
-or Extra-Criticals.
-
-	You cannot score Critical Hits while firing a gun in burst or automatic mode.
+
+	When you perform an Accuracy Roll (while not firing a gun in burst or 
+automatic mode), you may roll high enough to use your Critical Hit Chance. 
+Critical Hit Chance is based on your Luck Stat, but can also be modified by 
+abilities, weapons, attachments, etc. 
+	If your Accuracy Roll is high enough, you activate your Critical Hit Chance.
+When this happens, you roll a second d10 to determine if you have performed a Critical 
+Hit, an Extra-Critical Hit, or, if you have less than average Luck, merely hit your enemy. 
+
+	A Critical Hit deals +20 damage, while an Extra-Critical Hit deals +30 damage. 
+
+	Some abilities or feats have additional effects that trigger when you score 
```

Comment:
From a tone perspective, I'm not a great fan of "score." I suggest either using "perform" (as above), or changing both to "land" (which has combat connotations).